holoviews/operation/datashader.py

```diff
@@ -332,3 +332,70 @@ def _process(self, element, key=None):
         agg = aggregate._process(self, element, key)
         shaded = shade._process(self, agg, key)
         return shaded
+
+
+
+class dynspread(ElementOperation):
+    """
+    Spreading expands each pixel in an Image based Element a certain
+    number of pixels on all sides according to a given shape, merging
+    pixels using a specified compositing operator. This can be useful
+    to make sparse plots more visible. Dynamic spreading determines
+    how many pixels to spread based on a density heuristic.
+    """
```

Comment:
Maybe point to this section of the datashader manual, to pass the buck for having to explain it in any more detail? (Not that the datashader site necessarily does, just that if someone wants more explanation, that's the place it should be provided, not in holoviews.)